### ISSUE TITLE
Question: use for other site

### ISSUE THREAD

**Issue by ryuliantoro** [mymi14s/frappe_system_monitor#2]
Can we use this app to monitor other sites?
So only administrator of bench can monitor from an admin site.
If possible, what part of code need to be adjusted?
Thank you

**Comment by mymi14s** [comment]
Of course, you can modify the app to monitor other site, the code is open source and available for you to edit.
________________________________
From: Rah Yuliantoro ***@***.***>
Sent: Thursday, October 6, 2022 2:34 PM
To: mymi14s/frappe_system_monitor ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [mymi14s/frappe_system_monitor] Question: use for other site (Issue #2)


Can we use this app to monitor other sites?
So only administrator of bench can monitor from an admin site.
If possible, what part of code need to be adjusted?
Thank you
